Enhance QV tutorial and docs

ShellyGarion opened the pull request:
<!--
⚠️ If you do not respect this template, your pull request will be closed.
⚠️ Your pull request title should be short detailed and understandable for all.
⚠️ Also, please add it in the CHANGELOG file under Unreleased section.
⚠️ If your pull request fixes an open issue, please link to the issue.

✅ I have added the tests to cover my changes.
✅ I have updated the documentation accordingly.
✅ I have read the CONTRIBUTING document.
-->

### Summary



### Details and comments

ShellyGarion commented:
currently the batch experiments in the tutorial are not printing - @gadial - could you please review & fix it? thanks!

ShellyGarion commented:
Another issue: in cell no. 3:
`qv_exp.set_experiment_options(trials=60)`
does not effect the number of trials (it stays 100 instead of 160)

ShellyGarion commented:
If we can correct the notebook, then it may be interesting to find some `seed` such that with `qubits = 4`:
In cell 2 with `trials = 100` we have `QV = 1`
and in cell 3 with `trials = 160` we have `QV = 16`

chriseclectic commented:
Printing the full heavy output probabilities list in every result cell looks pretty ugly in the notebook (and that info is shown in the figure anyway)

coruscating commented:
But it's good for users to see how to get the full HOP list at least once. Maybe show it once and then only print mean HOP for the other runs?

ShellyGarion commented:
I think that it's better to see the full hop in case someone would like to further analyze it, but feel free to change this in another PR.
(I didn't change the code itself, just updated the documentation and added some further explanation)
